SLA.md

```diff
@@ -2,14 +2,16 @@
 ### Acquisition and learning
-Acquisition and learning are terms that can be strictly divided. Acquisition, according to Yule, is a process of gradual development in language by its natural usage. Moreover, it is present without help of a teacher. However, when it comes to learning, it is connected with a conscious action (or process) of expanding knowledge. Learning is typically visible in schools.
+Acquisition and learning are terms that can be strictly divided. Acquisition, according to Yule, is a process of gradual development in language by its natural usage. Moreover, it is present without the help of a teacher. However, when it comes to learning, it is connected with a conscious action (or process) of expanding knowledge. Learning is typically visible in schools.
 What is more, Yule claims that the traditional way of learning includes more knowledge considering the theory about language and not knowing about how to use it. People who acquired L2 in this traditional way of learning tend to be less effective in their language skills in comparison to those who acquired L2 in more ‘natural’ circumstances.
 ### Acquisition barriers
 The author introduces this issue explaining that most people’s experience with L2 begins in school when they have already acquired and got used to the sound structures of their L1. This problem might be demotivating for many but some of such learners, with enough commitment, develop their L2 skills. However, it is highly probable that they still will not resemble native speakers. The issue of having an ‘accent’, as Yule mentions, is common also for people that acquired L2 in perfect environments.
-### Affective factors
-
+### The Age Factor
+There is a view that “an inherent capacity for language” has got its strongest period before puberty and the acquisition of a new L2 is significantly harder after this time. On the other hand, an opposing view illustrates teenagers as more effective L2 learners in comparison to their younger colleagues (e.g. seven-year-olds). Thus, Yule suggests that proper learning of L2 requires a set of factors such as proper age (from ten to sixteen).
+### Affective Factors
+There is a possibility that the aforementioned optimum age for learning the second language will be blocked by another issue. As teenagers are more self-conscious, there might be a possibility for them to get embarrassed and, eventually, to get reluctant. Yule describes such negative facets as affective factors creating a barrier between the learner and the second language. One’s mentality is crucial in effective learning and cannot be unmentioned. Affective factors are less serious when it comes to younger learners and this might be why many researchers claim that age matters so much in terms of L2 learning. 
```

Comment:
Ja bym dała "become" przy reluctant, a tak to wszystko dobrze moim zdaniem :))

Reply:
Robi się!